src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -71,15 +71,9 @@ def setup(self, builder: Builder):
         self.population_view = builder.population.get_view(self.columns_used)
         self.businesses = None
 
-        job_change_rate_data = builder.lookup.build_table(data_values.YEARLY_JOB_CHANGE_RATE)
-        self.job_change_rate = builder.value.register_rate_producer(
-            f"{self.name}.job_change_rate", source=job_change_rate_data
-        )
+        self.job_change_rate = builder.lookup.build_table(data_values.YEARLY_JOB_CHANGE_RATE)
```

Comment:
Was something breaking or being processed incorrectly when using pipelines instead of lookup tables?

Reply:
no; they accomplish the same thing. however, pipelines allow you to register pipeline modifiers -- which we don't have or expect to have, so pipelines were overkill for our use-case